_posts/2020-08-31-community-update.md

```diff
@@ -0,0 +1,222 @@
+---
+layout: post
+title: "Flink Community Update - August'20"
+date: 2020-08-31T08:00:00.000Z
+categories: news
+authors:
+- morsapaes:
+  name: "Marta Paes"
+  twitter: "morsapaes"
+
+excerpt: Ah, so much for a quiet August month. This time around, we bring you some new Flink Improvement Proposals (FLIPs), a preview of the upcoming Flink Stateful Functions 2.2 release and a look into how far Flink has come in comparison to 2019.
+---
+
+Ah, so much for a quiet August month. This time around, we bring you some new Flink Improvement Proposals (FLIPs), a preview of the upcoming [Flink Stateful Functions](https://ci.apache.org/projects/flink/flink-statefun-docs-master/) 2.2 release and a look into how far Flink has come in comparison to 2019.
+
+{% toc %}
+
+# The Past Month in Flink
+
+## Flink Releases
+
+### Getting Ready for Flink Stateful Functions 2.2
+
+The details of the next release of [Stateful Functions](https://ci.apache.org/projects/flink/flink-statefun-docs-master/) are under discussion in [this @dev mailing list thread](http://apache-flink-mailing-list-archive.1008284.n3.nabble.com/Next-Stateful-Functions-Release-td44063.html), and the feature freeze is set for **September 10th** — so, you can expect Stateful Functions 2.2 to be released soon after! Some of the most relevant features in the upcoming release are:
+
+* **DataStream API interoperability**, allowing users to embed Stateful Functions pipelines in regular [DataStream API](https://ci.apache.org/projects/flink/flink-docs-stable/dev/datastream_api.html) programs with `DataStream` ingress/egress.
+
+* **Fine-grained control over state** for remote functions, including the ability to configure different state expiration modes for each individual function.
+
+As the community around StateFun grows, the release cycle will follow this pattern of smaller and more frequent releases to incorporate user feedback and allow for faster iteration. If you’d like to get involved, we’re always looking for [new contributors](https://github.com/apache/flink-statefun#contributing)!
+
+### Flink 1.10.2
+
+The community has announced the second patch version to cover some outstanding issues in Flink 1.10. You can find a detailed list with all the improvements and bugfixes that went into Flink 1.10.2 in the [announcement blogpost](https://flink.apache.org/news/2020/08/25/release-1.10.2.html).
+
+<hr>
```

Comment:
You could have mentioned the Flink 1.11.2 plans as well (or did this discussion start after you wrote the blog?

Reply:
Yeah, it started after I opened the PR. But for minor releases I usually only include the release itself once it's out.

Reply:
okay, I see.